calliope/core/model.py

```diff
@@ -125,9 +126,19 @@ def _init_from_model_run(self, model_run, debug_data):
             name='run_config', observer=self._model_data
         )
 
+        # Attach _model_run and _debug_data to _model_data
+        _model_run_to_save = self._model_run.copy()
+        del _model_run_to_save['timeseries_data']  # Can't be serialised!
+        self._model_data.attrs['_model_run'] = _model_run_to_save.to_yaml()
```

Comment:
I would have preference for `_model_run` only being added into the attrs at the point where a user saves the model to netcdf, i.e. you never see it in `inputs` or `results`, and it is never actually attached to `_model_data`, only to a copy of it when it is saved...

Reply:
Done